bearer/bearer_test.go

```diff
@@ -220,11 +220,11 @@ func TestToken_AssertContainer(t *testing.T) {
 
 	eaclTable := eacltest.Table()
 
-	eaclTable.SetCID(cidtest.ID())
+	eaclTable.LimitToContainer(cidtest.ID())
```

Comment:
The previous name was better. It shows, you are setting the ID, not a complete container

Reply:
for me it's not a problem. Usually u state "in a container <id>". But ok i'll change